Tests/XcodeProjTests/Extensions/AEXML+XcodeFormatTests.swift

```diff
@@ -121,6 +131,30 @@ class AEXML_XcodeFormatTests: XCTestCase {
         )
     }
 
+    func test_Scheme_attributes_sorted_when_original_sorted() {
+        validateAttributes(
+            expectedXML: expectedSchemeXml.cleaned,
+            childName: "Scheme",
+            attributes: [
+                "LastUpgradeVersion": "1320",
+                "wasCreatedForAppExtension": "YES",
+                "version": "1.7"
+            ]
+        )
+    }
+
+    func test_Scheme_attributes_sorted_when_original_unsorted() {
```

Comment:
minor: this test may be redundant, the order in which dictionary elements are specified to my knowledge is opaque to the underlying types, the dictionaries are equal between this test and `test_Scheme_attributes_sorted_when_original_sorted`, as such only one of those tests will suffice.

An interesting test could be the following (but may be overkill for this)

- Start with an unsorted raw XML (string) scheme
- Deserialize it (read it vis `XCScheme`)
- Serialize it back to XML
- Verify the correct sort order